_posts/tutorials/community/2023-01-14-mtls-with-javalin-ssl.md

```diff
@@ -153,7 +153,8 @@ libraryDependencies += "io.javalin.community.ssl" % "ssl-plugin" % "{{site.javal
 To secure our Javalin application with mTLS, we just need to load the certificates and private keys into the SSL plugin, and register it with our Javalin application.
 
 {% capture java %}
-SSLPlugin plugin = new SSLPlugin(conf -> {
+// import io.javalin.community.ssl.SslPlugin;
```

Comment:
I would like to add an information, where the class can be found.
As SSLPlugin was invalid, I did not find any information, where the class can be (non existing class).

Would you like to add a section or a hint at an other position?
Should I remove it here?